dpctl-capi/source/dpctl_sycl_device_selector_interface.cpp

```diff
@@ -82,8 +82,13 @@ __dpctl_give DPCTLSyclDeviceSelectorRef DPCTLCPUSelector_Create()
 __dpctl_give DPCTLSyclDeviceSelectorRef
 DPCTLFilterSelector_Create(__dpctl_keep const char *filter_str)
 {
+#if __SYCL_COMPILER_VERSION < 20210925
```

Comment:
Do we need this if-def or should just supporting 2021.3 and up be enough?

Reply:
We need it. The change happened between 2021.3 and 2021.4

Reply:
aha right. OK